server/models/user.js

```diff
@@ -8,22 +8,28 @@ const auth = require('./../constants/auth');
 const userSchema = mongoose.Schema({
     email: {
         type: String,
-        required: true,
         trim: true,
+        required: [ true, 'Email is required'],
         unique: 1
     },
     password: {
         type: String,
-        required: true,
-        minlength: 6
+        required: [ true, 'Password is required'],
+        minlength: [ 6, 'Should be at least 6 chars']
     },
     name: {
         type: String,
-        maxlength: 100
+        trim: true,
+        required: [ true, 'Name is required'],
+        minlength: [ 3, 'Should be at least 3 chars'],
```

Comment:
The username can be less than 3 chars.
I think, it is enough to mark this field as required and add maxlength validation.

Reply:
Well, I agree... Just wanted to practice different types of verification...
I think I would change it to 2... It is a strange name which is less than 2 chars, no???

Reply:
Also, I remember you told me that is a good idea to ensure that digits are not present in the name and last name... I think it would be also nice to add it here...

Reply:
Finally, I have added also a "no-digits" custom verification just for fun, to practice more ways to verify forms... It is pretty simple, actually! :D